### PR TITLE
lesson02/grid_printer.py, fizz_buzz.py

### PR DESCRIPTION
**Pull request by 22e88**
My first working version, didn't look at the solutions of the other students until now :) 
I can imagine there are lot's of other possible solutions... please give me feedback about what might be better approaches... :) Thank you!

### CODE REVIEW

File: students/elmar_m/lesson02/grid_printer.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+def gprint(number, unit):
+    '''
+    print a grid with <number> rows and
+    columns and <unit> width and height.
+    '''
+
+    aPart = '+' + unit * '-'
```
Comment (natasha-aleksandrova) [inline students/elmar_m/lesson02/grid_printer.py]: per python style guide, please use snake_case style for variables

**Comment by natasha-aleksandrova** [comment]
@22e88 please be sure to submit this assignment in EdX to get the grade

**Comment by 22e88** [comment]
I added lesson02 grid_printer exercise part 1 and part 2 now, and changed part 3 variable names to snake_case.

**Comment by 22e88** [comment]
Added lesson02/fizz_buzz.py
It seems that there's no possibility to create a separate pull request for me at the moment, so again i add the new stuff to pull request #54 and rename that to lesson02/grid_printer.py, fizz_buzz.py.

**Comment by natasha-aleksandrova** [comment]
Thank you, Fizz Buzz looks good!

**Pull request merged**